### ISSUE TITLE
[net] ifconfig??

### ISSUE THREAD

**Issue by Mellvik** [ghaerr/elks#1148]
This is an enhancement suggestion.

Now that we're running more than one ELKS system on a net, the need for something like `ifconfig` arises. Not primarily for configuring an interface but for querying it. Some times (more frequently than I'd like to admit) I ask myself what the address of my system is. There are many ways to fix this, running `ps` is one of them, putting the host name in the `sh` prompt is another.  But the `ps` route may be temporary since we may soon be in a situation in which the `ktcp` command line does not have literal IP addresses.

Thus the need for an `ifconfig` like command which initially could be just for reporting the address, net mask, gateway, type (slip/ether) and - possibly - access to Interface ioctls and low level statistics.

At a later stage it may be practical to add the ability to change the address (or netmask or gw) - even the MAC address.

--M

**Comment by ghaerr** [comment]
Rather than adding yet another executable, which takes up more valuable disk space (especially without using compressed executables), how about adding the desired information display to netstat's output?

**Comment by Mellvik** [comment]
@ghaerr, 

yes, it's worth considering, and actually that's where I started too. With an option to netstat in order to get only what we want, and not the whole shebang. And/Or possibly a different name and a hardlink. 
If the actual query (or operation) doesn't require contacting ktcp, it would speed it up a lot, in particular of net activity is ongoing.

In my thinking I moved on to a separate program when considering the potential to set various values, but of course netstat could to that too. 

Thank you!

—M
